company/team/index.md

```diff
@@ -342,6 +342,14 @@ To add yourself to this page, copy the following template, paste it at the end o
 - [stefan@sourcegraph.com](mailto:stefan@sourcegraph.com), [GitHub](https://github.com/stefanhengl)
 - Stefan lives in Berlin with his wife and two children. Prior to Sourcegraph, Stefan was a software engineer at Zalando and SAP, where he worked on data pipelines and low-latency, high-throughput microservices. He graduated in Physics but eventually realized that software engineering is his true passion. In his free time, Stefan enjoys cooking, plays chess with his daughter, and dreams about building a house in the wilderness of Norway.
 
+
+## Kimberly Hopson (she/her)
+
+- Developer Communications
+- Berwyn, PA
+- [kimberly@sourcegraph.com](mailto:kimberly@sourcegraph.com), other social media links (if any)
```

Comment:
I think you can just straight up omit the "other social media links" placeholder if you're not adding any, like:

```suggestion
- [kimberly@sourcegraph.com](mailto:kimberly@sourcegraph.com)
```